### ISSUE TITLE
Keyboard naviagation

### ISSUE THREAD

**Issue by tzezar** [tzezar/datagrid#4]
Partially implemented. It is possible to move around the cells with arrows and style the cells according to the current field. There remains the issue of accessibility and the fact that in such cells there may be some interactive element like a checkbox which should be able to be focused. Another question is expandable rows - should it be possible to move on them with arrows?
